kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/model/spec/ServerTemplate.java

```diff
@@ -241,4 +241,48 @@ public int hashCode() {
         result = 31 * result + (capabilities != null ? capabilities.hashCode() : 0);
         return result;
     }
+
+    public ServerTemplate cloneServerTemplate() {
+        synchronized (LOCK) {
+            ServerTemplate serverTemplate = new ServerTemplate();
+            serverTemplate.setId(this.getId());
+            serverTemplate.setName(this.getName());
+            containersSpec.forEach(c -> serverTemplate.addContainerSpec(new ContainerSpec(c.getId(), c.getContainerName(), c.getServerTemplateKey(), c.getReleasedId(), c.getStatus(), cloneContainerConfigMap(c.getConfigs()))));
+
+            final Map<Capability, ServerConfig> cloneServerConfig = new HashMap<>();
+            configs.entrySet().stream().forEach(entry -> cloneServerConfig.put(entry.getKey(), entry.getValue()));
```

Comment:
This can be still made much simpler using `final Map<Capability, ServerConfig> cloneServerConfig = new HashMap<>(configs);`. The result will be the same and it's much easier to read.